src/pages/Dashboard.tsx

```diff
@@ -1,35 +1,188 @@
         outcomeNumber: 21,
         keyStat: '21% increase per annum',
         keyNumber: '21%',
         tldr: 'string string string string string string string string string ',
-        takeaways: 'string string string string string string string string'
+        takeaways: 'string string string string string string string string',
+        requests: ['RequestId1'],
+        collaborators: ['UserId1']
     },
-    {
+    "insightId2": {
+        createdBy: 'userId2',
         ticketId: 'DATASCI-2202:',
-        title: 'Title blah blah blah',
-        date: '8/3/2010',
+        title: 'DEMO - Title blah blah blah',
+        createDate: 3,
+        lastUpdated: 4,
         imgs: [{src: Logo, alt: 'google image'}],
-        outcomeNumber: 1,
+        outcomeNumber: 25,
         keyStat: '25% increase per annum',
         keyNumber: '25%',
         tldr: 'string string string string string string string string string ',
-        takeaways: 'string string string string string string string string'
+        takeaways: 'string string string string string string string string',
+        requests: ['RequestId2'],
+        collaborators: ['UserId2']
+    }
+}
+
+const demoRequests: {[id: string]: FiledRequest} = {
+    "requestId1": {
+        createdBy: "BigBossId",
+        createDate: 123,
+        lastUpdated: 123,
+        title: "DEMO - Title blah blah blah",
+        assignee: "CodeMonkeyId",
+        priority: "High",
+        status: "Assigned",
+        deadline: 123
     },
-]
+    "requestId2": {
+        createdBy: "BigBossId",
+        createDate: 123,
+        lastUpdated: 123,
+        title: "DEMO - Title blah blah blah",
+        assignee: "CodeMonkeyId",
+        priority: "High",
+        status: "Assigned",
+        deadline: 123
+    }
+}
 
 const Dashboard = () => {
+    const [displayedRequests, setDisplayedRequests] = useState<{[id: string]: FiledRequest}>(demoRequests);
+    const [displayedInsights, setDisplayedInsights] = useState<{[id: string]: Insight}>(demoInsights);
+
+    useEffect(() => {
+        (async () => {
+            try {
+                const storedRequests = await getFiledRequests(TEST_ORGANIZATION, TEST_GROUP);
+                // TODO: Refactor, current state is a bit hacky to force React to update
+                setDisplayedRequests({...displayedRequests, ...storedRequests});
+            } catch (error) {
+                console.log("Fetching requests broken very sad")
+            }
+            try {
+                const storedInsights = await getInsights(TEST_ORGANIZATION, TEST_GROUP);
+                // TODO: Refactor, current state is a bit hacky to force React to update
+                setDisplayedInsights({...displayedInsights, ...storedInsights});
+            } catch (error) {
+                console.log("Fetching insights broken very sad")
+            }
+        })();
+    }, [])
+
+    // Generate and store hardcoded insight object for now
+    const createNewInsight = async () => {
```

Comment:
Don't need the async here, there's no request being sent

Reply:
Can you expand on this point, for this function we are writing to firestore